packages/roosterjs-content-model/lib/domToModel/context/createDomToModelContext.ts

```diff
@@ -59,8 +59,8 @@ export function createDomToModelContext(
         defaultFormatParsers: defaultFormatParsers,
     };
 
-    if (options?.allowCacheElement) {
-        context.allowCacheElement = true;
+    if (typeof options?.disableCacheElement == 'boolean') {
+        context.disableCacheElement = options?.disableCacheElement;
```

Comment:
Actually, I don't think we need to change context. Just change the options is ok.

Reply:
Removed the changes to the Context interface.